### PR TITLE
RATEST-90 :failing Test: Add Patient Appointment Test

### PR DESCRIPTION
**Pull request by insookwa**
https://issues.openmrs.org/browse/RATEST-90

Test Passes now
please kindly review 
 cc @k-joseph  @kdaud , @sherrif10

### CODE REVIEW

**Review by sherrif10 — APPROVED** [review]
Well done @insookwa  and wel be back again
LGTM after testing it locally cc @k-joseph

**Comment by kdaud** [comment]
> LGTM after testing it locally

@sherrif10 are you sure the changes passes with CI ?

Am skeptical about what `CI report` will produce concerning the changes !

**Comment by sherrif10** [comment]
Like i commented above this test pass locally , we can only track other results via github actions  but since he is the first committer, he needs to be approved then github actions can get activated on this pull request

**Comment by kdaud** [comment]
> this test pass locally

IMO Its a good idea `PR approvals` keep on hold until we can clearly see the CI report `(green)` on a test. 

> he needs to be approved then github actions can get activated on this pull request

Have raised this to the community [here](https://talk.openmrs.org/t/ci-build-plan-on-openmrs-modules-denies-first-time-contributors/33405/2)

File: pom.xml [hunk 1]
```diff
@@ -40,37 +40,37 @@
...
 		<metadatadeployVersion>1.13.0</metadatadeployVersion>
-		<metadatasharingVersion>1.9.0-SNAPSHOT</metadatasharingVersion>
+		<metadatasharingVersion>1.8.0</metadatasharingVersion>
 		<metadatamappingVersion>1.4.0</metadatamappingVersion>
 		<uicommonsVersion>2.18.0</uicommonsVersion>
 		<appuiVersion>1.14.0</appuiVersion>
 		<htmlformentryVersion>4.0.1</htmlformentryVersion>
 		<htmlformentryuiVersion>2.0.0-SNAPSHOT</htmlformentryuiVersion>
-		<coreappsVersion>1.31.2-SNAPSHOT</coreappsVersion>
+		<coreappsVersion>1.32.0</coreappsVersion>
```
Comment (sherrif10) [inline pom.xml:55]: Thanks @insookwa Please remove un wanted commits

File: pom.xml [hunk 2]
```diff
@@ -40,37 +40,37 @@
 		<referenceapplicationVersion>2.12.0-SNAPSHOT</referenceapplicationVersion>
 		<providermanagementVersion>2.13.0</providermanagementVersion>
 		<uilibraryVersion>2.0.7</uilibraryVersion>
-		<calculationVersion>1.3.0-SNAPSHOT</calculationVersion>
+		<calculationVersion>1.3.0</calculationVersion>
```
Comment (sherrif10) [inline pom.xml:43]: To revert last commit, you can say **git log** in your terminal , the copy the first commit id which is the last on github
then say  **git revert <last commit id>.** after push

**Comment by insookwa** [comment]
fixing as recommended @sherrif10

**Pull request closed without merging**